ls1intum/Artemis
# `Development`: Delete duplicated Hermes service documentation

### Checklist
#### General


#### Server

### Motivation and Context


The Hermes Service documentation was duplicated word for word in the documentation.
### Description

I deleted the one of the duplicates

### Steps for Testing

Go to https://ls1intum.github.io/Artemis/dev/setup/#hermes-service and see that on the left hand menu, another Hermes Service entry exists with the exact same content.
Look at the new documentation (https://artemis-platform--7155.org.readthedocs.build/en/7155/dev/setup.html#hermes-service), and confirm that Hermes Service is now not duplicated anymore.

### Screenshots
#### Before
![Screenshot 2023-09-08 at 09 20 48](https://github.com/ls1intum/Artemis/assets/16179620/16f0171a-cb86-4558-8118-ef048bf42cb2)
#### Now
![Screenshot 2023-09-08 at 09 33 43](https://github.com/ls1intum/Artemis/assets/16179620/667f458d-93ed-4898-a8c1-44ff327ffa4d)
